tutorials/polkadot-sdk/build-a-parachain/connect-a-parachain.md

```diff
@@ -0,0 +1,331 @@
+         <br>
+         <span data-ty="progress">2024-09-10 14:41:13 🔨 Initializing Genesis block/state (state: 0xb089…1830, header-hash: 0x6b0b…bd69)</span>
+      </div>
+
+    !!!note
+        You should note that the runtime and state you export must be for the genesis block. You can't connect a parachain with any previous state to a relay chain. All parachains must start from block 0 on the relay chain. See [Convert a Solo Chain](https://docs.substrate.io/reference/how-to-guides/parachains/convert-a-solo-chain/){target=\_blank} for details on how the parachain template was created and how to convert the chain logic—not its history or state migrations—to a parachain.
+
+3. Start a collator node with a command similar to the following
+
+    ```bash
+    ./target/release/parachain-template-node \
+      --charlie \
+      --collator \
+      --force-authoring \
+      --chain raw-parachain-chainspec.json \
+      --base-path /tmp/charlie-parachain/ \
+      --unsafe-force-node-key-generation \
+      --port 40333 \
+      --rpc-port 8844 \
+      -- \
+      --chain <INSERT_RELAY_CHAIN_PATH>/local-raw-spec.json \
+      --port 30333 \
+      --rpc-port 9946
+    ```
+
+    !!! note
+        Ensure that you replace `<INSERT_RELAY_CHAIN_PATH>` with the path to the raw chain specification for the local relay chain.
+
+    After running the command, you will see the following output:
+
+    <div id="termynal" data-termynal>
+    <span data-ty="input"><span class="file-path"></span>./target/release/parachain-template-node \
+      --charlie \
+      --collator \
+      --force-authoring \
+      --chain raw-parachain-chainspec.json \
+      --base-path /tmp/charlie-parachain/ \
+      --unsafe-force-node-key-generation \
+      --port 40333 \
+      --rpc-port 8844 \
+      -- \
+      --chain <INSERT_RELAY_CHAIN_PATH>/local-raw-spec.json \
+      --port 30333 \
+      --rpc-port 9946
+    </span>
+    <span>...</span>
+    <br>
+    <span>2024-09-10 16:26:30 [Parachain] PoV size { header: 0.21875kb, extrinsics: 3.6103515625kb, storage_proof: 3.150390625kb }</span>
+    <br>
+    <span>2024-09-10 16:26:30 [Parachain] Compressed PoV size: 6.150390625kb</span>
+    <br>
+    <span>2024-09-10 16:26:33 [Relaychain] 💤 Idle (2 peers), best: #1729 (0x3aa4…cb6b), finalized #1726 (0xff7a…4352), ⬇ 9.1kiB/s ⬆ 3.8kiB/s</span>
+    <br>
+    <span>...</span>
+    </div>
+
+## Register With the Local Relay Chain
+
+With the local relay chain and collator node running, you can register the parachain on the local relay chain. In a live public network, registration typically involves a [parachain auction](https://wiki.polkadot.network/docs/learn-auction){target=\_blank}. You can use a Sudo transaction and the Polkadot.Js App interface for this tutorial and local testing. A Sudo transaction lets you bypass the steps required to acquire a parachain or parathread slot.
```

Comment:
Not sure if we should keep this and wait until core time is live, or just  add the explanation of how to get core time in here instead of getting a parachain slot

Reply:
afaik core time went live some days ago, right? However, I'm unsure if it's already prepared for this procedure. Do you have any input on that @0xLucca or @CrackTheCode016 ?

Reply:
I'm not sure. I have yet to try the local version. We can stick to this for now because it is a local network, and we can do it with core time when connecting to a live testnet